Doc/library/pkgutil.rst

```diff
@@ -127,7 +127,8 @@ support.
    Yields :class:`ModuleInfo` for all submodules on *path*, or, if
    *path* is ``None``, all top-level modules on :data:`sys.path`.
 
-   *path* should be either ``None`` or a list of paths to look for modules in.
+   *path* should be either ``None`` or a list of paths as strings to search
```

Comment:
An alternative here might be to specify the type in parens, which might be easier to read:
"...a list of paths (``List[str]``)..."
 "...a list of paths (where each path is  ``str``)..." 

...or similar. Not married to it, but I thought it might be worth considering.

Reply:
Seems reasonable. TBH, I was struggling to figure out the phrasing.